news/2020-06-26-mappers-report-May-June.md

```diff
@@ -0,0 +1,193 @@
+- **osu!taiko:** [realy0\_](https://osu.ppy.sh/users/8863359), [\_DUSK\_](https://osu.ppy.sh/users/6092181), [davidminh0111](https://osu.ppy.sh/users/9623142), [Cychloryn](https://osu.ppy.sh/users/6921736) & [Axer](https://osu.ppy.sh/users/7299864)
+- **osu!catch:** [Baron](https://osu.ppy.sh/users/10286499), [Secre](https://osu.ppy.sh/users/2306637) & [Bastian](https://osu.ppy.sh/users/6345176)
+- **osu!mania:** [Kyousukee](https://osu.ppy.sh/users/8842107), [Shima Rin](https://osu.ppy.sh/users/6089608), [Garalulu](https://osu.ppy.sh/users/757783) & [Muse Dash](https://osu.ppy.sh/users/13695676)
+
+### BN Resignations
+
+- **osu!:** [Affirmation](https://osu.ppy.sh/users/6186628) & [wafer](https://osu.ppy.sh/users/9416836)
+- **osu!taiko:** [Cynplytholowazy](https://osu.ppy.sh/users/3901754) & [Gamelan4](https://osu.ppy.sh/users/9856910)
+- **osu!catch:** [-Luminate](https://osu.ppy.sh/users/4778689)
+- **osu!mania:** [Drum-Hitnormal](https://osu.ppy.sh/users/748722), [\_Kobii](https://osu.ppy.sh/users/6209713) & [Unpredictable](https://osu.ppy.sh/users/7560872)
+
+### BN and QAT/NAT Badges
+
+Recently a new wave of badges acknowledging the length of time members have put into teams for managing ranked! Those who are on this list have been contributing to the community in so many ways during their time on osu!, and fully deserve this acknowledgement. We thank them all for their contribution so far and hope to see much more of them in the future! Here is the list of those:
+
+- **BN 1 Year Badge:** [incandescence](osu.ppy.sh/users/6256027), [Kaitjuh](osu.ppy.sh/users/2225327), [Unpredictable](osu.ppy.sh/users/7560872), [Nuvolina](osu.ppy.sh/users/10974170), [Dored](osu.ppy.sh/users/10284894)
+- **BN 2 Year Badge:** [JBHyperion](osu.ppy.sh/users/4879508), [Nevo](osu.ppy.sh/users/7451883), [Mordred](osu.ppy.sh/users/7265097), [bossandy](osu.ppy.sh/users/360437), [Namki](osu.ppy.sh/users/5248582)
+- **BN 3 Year Badge:** [Benita](osu.ppy.sh/users/4023183), [Sonnyc](osu.ppy.sh/users/11771)
+- **BN 4 Year Badge:**  [Nozhomi](osu.ppy.sh/users/2716981)
+- **NAT 1 Year Badges:** [Ascendance](osu.ppy.sh/users/2931883), [Noffy](osu.ppy.sh/users/1541323), [Faputa](osu.ppy.sh/users/845733)
+- **NAT 2 Year Badge:** [Lasse](osu.ppy.sh/users/896613), [-Mo-](osu.ppy.sh/users/2202163), [Chaoslitz](osu.ppy.sh/users/3621552)
+- **NAT 3 Year Badge:** [OnosakiHito](osu.ppy.sh/users/290128)
+
+## <a id="mappers-showcase"></a>Mappers' Showcase
+
+The Mappers' Showcase parades lesser-known mappers from every game mode. Since we bundled two issues in one post, this month will feature two game modes. 
+Thanks to our selector for osu!catch [Jemzuu](https://osu.ppy.sh/users/7890134) and our selectors for osu!mania [Scotty](https://osu.ppy.sh/users/11085809) & [Unpredictable](https://osu.ppy.sh/users/7560872)!
```

Comment:
Could mention "Phob and dika312" for catch and  "BringoBrango and Ticle" for mania so it's clear who's what mode being showcased

Reply:
Good idea, added game mode behind the names to make it more clear